updater/fetchers/nodejs/nodejs.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2016 clair authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nodejs
+
+import (
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/coreos/clair/database"
+	"github.com/coreos/clair/updater"
+	cerrors "github.com/coreos/clair/utils/errors"
+	"github.com/coreos/clair/utils/types"
+	"github.com/coreos/pkg/capnslog"
+)
+
+const (
+	url                  = "https://api.nodesecurity.io/advisories"
+	cveURLPrefix         = "http://cve.mitre.org/cgi-bin/cvename.cgi?name="
+	updaterFlag          = "nodejsUpdater"
+	defaultNodejsVersion = "all"
+	//FIXME: Add a suffix when an advisory is fixed `after` a certain version.
```

Comment:
I'll remove the 'FIXME'. I added 'FIXME' just to remind it needs a better way to check if a package is vulnerable. 

In current fetchers, if a package's version is recorded as '1.0.1', it means the security issue will be fixed when the package version '>=' 1.0.1.  But in nodejs, it is more complicated. Sometimes, it should '>=' 1.0.1, sometimes it should '>' 1.0.1.